packages/core/test/insured-bridge/e2e/InsuredBridge.e2e.js

```diff
@@ -47,67 +60,131 @@ describe("L1 <> L2 Deposit and Withdrawal", () => {
-      assert.equal((await L1_ERC20.balanceOf(l1Wallet.address)).toString(), "0");
-      assert.equal((await L2_ERC20.balanceOf(l2Wallet.address)).toString(), "1234");
+      assert.equal((await l1Erc20.balanceOf(l1Wallet.address)).toString(), "0");
+      assert.equal((await l2Erc20.balanceOf(l2Wallet.address)).toString(), "1234");
     });
     it("Can send tokens back from L2 to L1", async () => {
       // Burn the tokens on L2 and ask the L1 contract to unlock on our behalf.
-      const withdrawTx = await L2StandardBridge.withdraw(L2_ERC20.address, 1234, 0, "0x");
+      const withdrawTx = await l2StandardBridge.withdraw(l2Erc20.address, 1234, 0, "0x");
       await withdrawTx.wait();
 
       // Wait for the message to be relayed to L1. Check addresses in the logs match the L1 token and bridge.
-      const [msgHash2] = await watcher.getMessageHashesFromL2Tx(withdrawTx.hash);
-      const receipt = await watcher.getL1TransactionReceipt(msgHash2);
-      assert.equal(receipt.logs[0].address, L1_ERC20.address);
-      assert.equal(receipt.logs[1].address, L1StandardBridge.address);
+      const [msgHash] = await watcher.getMessageHashesFromL2Tx(withdrawTx.hash);
+      const receipt = await watcher.getL1TransactionReceipt(msgHash);
+      assert.equal(receipt.logs[0].address, l1Erc20.address);
+      assert.equal(receipt.logs[1].address, l1StandardBridge.address);
 
       // Validate balances on L1 have been incremented accordingly.
-      assert.equal((await L1_ERC20.balanceOf(l1Wallet.address)).toString(), "1234");
-      assert.equal((await L2_ERC20.balanceOf(l2Wallet.address)).toString(), "0");
+      assert.equal((await l1Erc20.balanceOf(l1Wallet.address)).toString(), "1234");
+      assert.equal((await l2Erc20.balanceOf(l2Wallet.address)).toString(), "0");
+    });
+  });
+  describe("Insured bridge functionality", async () => {
+    // Contract objects
+    let l1Finder, l1BridgeRouter, l2Timer, l2BridgeDepositBox;
+    beforeEach("Deploy and configure insured bridge contracts on appropriate networks", async () => {
+      let optimisticOracleLiveness = 7200;
+      let minimumBridgingDelay = 60;
+
+      // Deploy UMA helper contracts
+      l1Finder = await factory__L1_Finder.connect(l1Wallet).deploy();
+      l2Timer = await factory__L2_Timer.connect(l2Wallet).deploy(OPTIMISM_GAS_OPTS);
+      await l2Timer.deployTransaction.wait();
+
+      // TODO: when doing full integration tests we will need the OO as well. For now show that we can do basic bridging.
+
+      // Bridging infrastructure and initialization. Note we use the l1 proxy cross-domain messenger for the routers
+      // _crossDomainMessenger. This is done to mimic the production setup which routes transactions through this proxy.
+      l1BridgeRouter = await factory__L1_BridgeRouter
+        .connect(l1Wallet)
+        .deploy(l1Finder.address, PROXY__OVM_L1_CROSS_DOMAIN_MESSENGER, l1Wallet.address, optimisticOracleLiveness);
```

Comment:
Nice! The merge conflicts on this file will be minimal. Just changes to the contract name and the constructor signature.